io_stream.c

```diff
@@ -116,7 +158,11 @@ void* php_mongo_io_stream_connect(mongo_con_manager *manager, mongo_server_def *
 
 		zend_replace_error_handling(EH_THROW, mongo_ce_ConnectionException, &error_handler TSRMLS_CC);
 
+#if PHP_VERSION_ID >= 50600 && PHP_VERSION_ID < 50607
```

Comment:
Perhaps add a link to the bug report / PHP Commit here? - or at least an explanation why 5.6.0-5.6.7 are different in a comment?